src/main/java/Event.java

```diff
@@ -0,0 +1,45 @@
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class Event extends Task {
+    protected LocalDateTime eventDateTime;
+
+    public Event(String description, LocalDateTime dateTime) {
+        super(description);
+        this.eventDateTime = dateTime;
+    }
+
+    public Event(String description, boolean isDone, LocalDateTime dateTime) {
+        this(description, dateTime);
+        this.isDone = isDone;
+    }
+
+    /**
+     * Getter for when this Event occurs.
```

Comment:
Instead of writing getter, use the format specified [here under the section for javadocs.](https://nus-cs2103-ay1920s1.github.io/website/coding-standards/java/intermediate.html)